domains/yusuf-rawat.json

```
@@ -0,0 +1,10 @@
+{
+  "description": "this a link to my github profile.",
+  "owner": {
+    "username": "yusuf007r",
+    "email": "yusufmaster7@gmail.com"
+  },
+  "record": {
+    "CNAME": "github.com/Yusuf007R"
```

`CNAME` has to be a valid hostname. If you want to redirect to your github account, use URL record type.

```suggestion
    "URL": "https://github.com/Yusuf007R"
```